ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -88,15 +88,20 @@ conformance_test(
         ",ExceptionsIT,ExceptionRaceConditionIT" +  # need UCK mode - added below
         ",DeeplyNestedValueIT" +  # FIXME: Too deeply nested values flake with a time out (half of the time)
         ",CommandServiceIT:CSReturnStackTrace" +  # FIXME: Ensure canton returns stack trace
-        ",CommandDeduplicationIT:ParticipantCommandDeduplicationSimpleDeduplicationBasic,CommandDeduplicationIT:ParticipantCommandDeduplicationDeduplicateSubmitterBasic,CommandDeduplicationIT:ParticipantCommandDeduplicationSimpleDeduplicationMixedClients" +  # sync vs async error (part of canton #6301)
+        ",CommandDeduplicationIT:ParticipantCommandDeduplicationSimpleDeduplicationBasic" +  # sync vs async error (part of canton #6301)
+        ",CommandDeduplicationIT:ParticipantCommandDeduplicationDeduplicateSubmitterBasic" +
+        ",CommandDeduplicationIT:ParticipantCommandDeduplicationSimpleDeduplicationMixedClients" +
         # Also exclude "optional tests" - which are run separately below
         ",CompletionDeduplicationInfoITCommandService" +
         ",CompletionDeduplicationInfoITCommandSubmissionService" +
         ",CommandDeduplicationParallelIT" +
         ",ContractIdIT" +
         ",MultiPartySubmissionIT" +
         ",ParticipantPruningIT" +
-        ",KVCommandDeduplicationIT",  # only for KV-utils
+        ",TLSOnePointThreeIT" +
+        ",TLSAtLeastOnePointTwoIT" +
+        ",KVCommandDeduplicationIT" +  # only for KV-utils
```

Comment:
Not related to your change, but the glance of this in the Canton context is confusing.